README.md

```diff
@@ -150,6 +150,24 @@ The above steps will deploy the statically generated API docs by running the `np
     - dev environment: Singapore (`ap-southeast-1`)
     - prod environment: Mumbai (`ap-south-1`)
 
+## Github Actions
+### Steps for CI/CD :
+1. Store all account credentials in github secrets to access them in environment without exposing.
```

Comment:
```suggestion
1. Store all account credentials in GitHub [secrets](https://docs.github.com/en/actions/reference/encrypted-secrets) to access them in environment without exposing. The AWS credentials will be used for deploying the application into your AWS account and the database details is used for running the test cases as a part of the CICD setup
```

Reply:
Addressed this suggestion in latest commit